# Use EasyBuild environment variables
# Modify sed statement to consider only words ending with ".o"
# This caused troubles when using paths that included non-ending ".o"
#
# Davide Vanzo (Vanderbilt University)
diff -ru tandem-linux-17-02-01-4.orig/src/Makefile tandem-linux-17-02-01-4/src/Makefile
--- tandem-linux-17-02-01-4.orig/src/Makefile	2019-06-27 11:06:29.621489466 -0500
+++ tandem-linux-17-02-01-4/src/Makefile	2019-06-27 11:42:25.245549113 -0500
@@ -1,51 +1,50 @@
-#makefile for c++ programs
-#change the name of the executable to use for "any" project
-
-EXECUTABLE = ../bin/tandem.exe
-#EXECUTABLE = ../bin/p3.exe
-LINKCC = $(CXX)
-
-#CXXFLAGS denotes flags for the C++ compiler
-
-CXX = g++
-
-#uncomment this line if you are using gcc 4.x
-CXXFLAGS = -O2 -DGCC4_3
-#CXXFLAGS = -O2 -DGCC4_3 -DX_P3
-#CXXFLAGS = -O2 -DGCC4_3 -DPLUGGABLE_SCORING -DX_P3
-
-#LDFLAGS = -lpthread -L/usr/lib -lm /usr/lib64/libexpat.a -static-libstdc++
-LDFLAGS = -lpthread -L/usr/lib -lm /usr/lib64/libexpat.a
-#LDFLAGS = -lpthread -L/usr/lib -lm -lexpat
-SRCS := $(wildcard *.cpp)
-OBJS := $(patsubst %.cpp,%.o,$(wildcard *.cpp))
-DEPS := $(patsubst %.o,%.d,$(OBJS))
-
-all: $(EXECUTABLE)
-
-#define the components of the program, and how to link them
-#these components are defined as dependencies; that is they must be up-to-date before the code is linked
-
-$(EXECUTABLE): $(DEPS) $(OBJS)
-	$(LINKCC) $(CXXFLAGS) -o $(EXECUTABLE) $(OBJS) $(LDFLAGS)
-
-#specify the dep files depend on the cpp files
-
-%.d: %.cpp
-	$(CXX) -M $(CXXFLAGS) $< > $@
-	$(CXX) -M $(CXXFLAGS) $< | sed s/\\.o/.d/ > $@
-
-clean:
-	-rm $(OBJS) $(EXECUTABLE) $(DEPS) *~
-
-explain:
-	@echo "The following info represents the program:"
-	@echo "Final exec name: $(EXECUTABLE)"
-	@echo "Source files:       $(SRCS)"
-	@echo "Object files:       $(OBJS)"
-	@echo "Dep files:          $(DEPS)"
-
-depend: $(DEPS)
-	@echo "Deps are now up-to-date."
- 	
--include $(DEPS)
+#makefile for c++ programs
+#change the name of the executable to use for "any" project
+
+EXECUTABLE = ../bin/tandem.exe
+#EXECUTABLE = ../bin/p3.exe
+LINKCC := $(CXX)
+
+#CXXFLAGS denotes flags for the C++ compiler
+CXX := $(CXX)
+
+#uncomment this line if you are using gcc 4.x
+CXXFLAGS := $(CXXFLAGS)
+#CXXFLAGS = -O2 -DGCC4_3 -DX_P3
+#CXXFLAGS = -O2 -DGCC4_3 -DPLUGGABLE_SCORING -DX_P3
+
+#LDFLAGS = -lpthread -L/usr/lib -lm /usr/lib64/libexpat.a -static-libstdc++
+LDFLAGS := -lpthread -lexpat $(LDFLAGS)
+#LDFLAGS = -lpthread -L/usr/lib -lm -lexpat
+SRCS := $(wildcard *.cpp)
+OBJS := $(patsubst %.cpp,%.o,$(wildcard *.cpp))
+DEPS := $(patsubst %.o,%.d,$(OBJS))
+
+all: $(EXECUTABLE)
+
+#define the components of the program, and how to link them
+#these components are defined as dependencies; that is they must be up-to-date before the code is linked
+
+$(EXECUTABLE): $(DEPS) $(OBJS)
+	$(LINKCC) $(CXXFLAGS) -o $(EXECUTABLE) $(OBJS) $(LDFLAGS)
+
+#specify the dep files depend on the cpp files
+
+%.d: %.cpp
+	$(CXX) -M $(CXXFLAGS) $< > $@
+	$(CXX) -M $(CXXFLAGS) $< | sed s/\\.o\b/.d\b/ > $@
+
+clean:
+	-rm $(OBJS) $(EXECUTABLE) $(DEPS) *~
+
+explain:
+	@echo "The following info represents the program:"
+	@echo "Final exec name: $(EXECUTABLE)"
+	@echo "Source files:       $(SRCS)"
+	@echo "Object files:       $(OBJS)"
+	@echo "Dep files:          $(DEPS)"
+
+depend: $(DEPS)
+	@echo "Deps are now up-to-date."
+ 	
+-include $(DEPS)
Only in tandem-linux-17-02-01-4/src: Makefile~
